libwacom/libwacom.h

```diff
@@ -638,6 +648,12 @@ WacomAxisTypeFlags libwacom_stylus_get_axes (const WacomStylus *stylus);
  */
 WacomStylusType libwacom_stylus_get_type (const WacomStylus *stylus);
 
+/**
+ * @param stylus The stylus to query
+ * @return The type of eraser on the stylus
+ */
+WacomEraserType libwacom_stylus_get_eraser_type (const WacomStylus *stylus);
```

Comment:
I'm surprised this builds. You need to add this to the `libwacom.sym` files like this:
```
LIBWACOM_1.2 {
   libwacom_stylus_get_eraser_type;
} LIBWACOM_0.33;
```

Reply:
Looks like my update to that file didn't get added into git :man_facepalming: